src/blackbox_decode.c

```diff
@@ -483,6 +485,13 @@ void outputGPSFrame(flightLog_t *log, int64_t *frame)
     }
 }
 
+/**
+ * Get altitude in [m] from betaflight logged [cm], including optional user altitude offset.
+ */
+float getAltitude(flightLog_t *log, int64_t *frame) {
```

Comment:
You have to move this up before the first use of it, or else the compilation will fail (see the 'Checks' tab for details).